deployment/ccip/changeset/internal/deploy_home_chain.go

```diff
@@ -0,0 +1,536 @@
+package internal
```

Comment:
Just took one file to use as an example. There is a lot more we can hide in internal